src/element/Nip05.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import './Nip05.css'
+
+const Nip05 = ({ nip05, pubkey }) => {
+    const [nip05pubkey, setNip05pubkey] = useState()
+    const isVerified = nip05pubkey === pubkey
+    const [nick, domain] = nip05.split('@')
+
+    useEffect(() => {
+        fetch(`https://${domain}/.well-known/nostr.json`)
```

Comment:
You should include the param `?name=${nick}` some services will require this

Reply:
https://github.com/nostr-protocol/nips/blob/master/05.md

Reply:
Good point, done.

Reply:
Also treating `_` as the default user and following spec recommendation for display:
<img width="282" alt="Screenshot 2023-01-10 at 07 18 49" src="https://user-images.githubusercontent.com/92299987/211476434-d9eae9e0-aa52-423a-8d78-26217bcfdaa2.png">
<img width="282" alt="Screenshot 2023-01-10 at 07 18 44" src="https://user-images.githubusercontent.com/92299987/211476475-79e03d9d-4224-4ddf-8c63-62e2e75e11ef.png">